docs/source/conf.py

```diff
@@ -21,7 +21,7 @@
 # -- Project information -----------------------------------------------------
 
 project = u'ESP Easy'
-copyright = u'2018-2022, ESP Easy'
+copyright = u'2018-2023, ESP Easy'
```

Comment:
So now we must wait to merge this PR? ;)

Reply:
Well, we're already quite close, so it doesn't matter much if we're a bit early 😃